internal/driver/webhtml.go

```diff
@@ -511,6 +615,11 @@ function dotviewer(nodes) {
     return null
   }
 
+  // convert a string to a regexp that matches that string.
+  function quotemeta(str) {
+    return str.replace(/([\\\.?+*\[\](){}])/g, '\\$1')
```

Comment:
I selected a function that showed "std::abs (inline)" and hit "show", and the profile came back empty. I think it we should strip the "(inline)" part for the selected expressions (or even better have it on a separate column.